envs/example/standard/hosts

```diff
@@ -17,8 +17,10 @@ compute1
 compute1
 
 [ceph_monitors]
+# when expanding cluster, place new hosts BELOW existing hosts
```

Comment:
I'm not sure the best way to enforce this w/ ansible. hopefully this works for MVP..